go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/digitalbitbox/bitbox-wallet-app
 
-go 1.14
+go 1.16
```

Comment:
Also FYI here: makes sense to bump only if the plan is to use newer features of go introduced in 1.16. https://golang.org/doc/modules/gomod-ref

Reply:
What about potential security updates?

Reply:
those are covered by patch releases: https://golang.org/doc/devel/release#go1.14.minor